packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/saved_objects_importer.ts

```diff
@@ -57,40 +62,56 @@ export class SavedObjectsImporter implements ISavedObjectsImporter {
     compatibilityMode,
     managed,
   }: SavedObjectsImportOptions): Promise<SavedObjectsImportResponse> {
-    return importSavedObjectsFromStream({
-      readStream,
-      createNewCopies,
-      namespace,
-      overwrite,
-      refresh,
-      compatibilityMode,
-      objectLimit: this.#importSizeLimit,
-      savedObjectsClient: this.#savedObjectsClient,
-      typeRegistry: this.#typeRegistry,
-      importHooks: this.#importHooks,
-      managed,
-    });
+    this.#log.debug('Starting the import process');
+    try {
+      const result = await importSavedObjectsFromStream({
```

Comment:
Awaiting a resolved promise from `importSavedObjectsFromStream` is a problem when importing many saved objects and can cause Kibana to run out of memory. 
import shouldn't await on `importSavedObjectsFromStream`, we have to stick with calling the method directly.

I'd rather move the log line to before returning `importSavedObjectsFromStream` and revert the other changes here,

Reply:
See [node streams](https://nodejs.org/api/stream.html#streams-promises-api) for more info